mods/cnc/rules/campaign-tooltips.yaml

```diff
@@ -42,6 +42,14 @@
 		GenericVisibility: Enemy, Neutral
 		ShowOwnerRow: false
 
+^CivHaystack:
```

Comment:
Doesn't this need to be `^CivHaystackOrIgloo` as well now?

Reply:
Yes.